ARCHITECTURE.md

```diff
@@ -163,8 +163,9 @@ using [`xtask`] in [codegen/gen_syntax_nodes]. We also generate
 `SyntaxKind`([codegen/gen_syntax_kinds]) that we use when creating nodes in the
 untyped syntax tree. 
 
-To illustrate how a definition in `ungrammar` becomes a struct, let's lookGiven the definition, we can then generate a struct with applicable accessor
-methods for `Description`, scalar token, `Name` and `Directives`.
+To illustrate how a definition in `ungrammar` becomes a struct, let's look at `ScalarTypeDefinition` 
+below. Given the definition, we can then generate a struct with applicable accessor methods for 
+`Description`, scalar token, `Name` and `Directives`.
```

Comment:
```suggestion
Given the definition, we can then generate a struct with applicable accessor methods for `Description`, 
scalar token, `Name` and `Directives`.
```


just came across [this blog post](https://www.apollographql.com/blog/announcement/tooling/apollo-rs-graphql-tools-in-rust/); probs want the wording to be consistent